Source/Blazorise/Models/MessageOptionsChoice.cs

```diff
@@ -0,0 +1,79 @@
+﻿#region Using directives
+using System;
+#endregion
+
+namespace Blazorise;
+
+/// <summary>
+/// Represents a command for a choice message, containing properties for a key, display text, icon, icon color, and overall color.
+/// </summary>
+public class MessageOptionsChoice
+{
+    /// <summary>
+    /// A default constructor for the <see cref="MessageOptionsChoice"/> class.
+    /// </summary>
+    public MessageOptionsChoice()
+    {
+        Key = Guid.NewGuid();
+    }
+
+    /// <summary>
+    /// Constructs a new instance with specified text and color for a message option.
+    /// </summary>
+    /// <param name="text">The content that will be displayed as the message option.</param>
+    /// <param name="color">The color that will be used to represent the message option visually.</param>
+    public MessageOptionsChoice( string text, Color color )
+        : this()
+    {
+        Text = text;
+        Color = color;
+    }
+
+    /// <summary>
+    /// Constructs a new instance with specified text and color for a message option.
+    /// </summary>
+    /// <param name="key">The unique key for the result of the choice.</param>
+    /// <param name="text">The content that will be displayed as the message option.</param>
+    /// <param name="color">The color that will be used to represent the message option visually.</param>
+    public MessageOptionsChoice( object key, string text, Color color )
```

Comment:
optional parameters here and keeping it in one method? (that way you will also add overload where only the key and text is specified)

Reply:
I'm not sure if I understand. Can you elaborate?

Reply:
something like

```
public MessageOptionsChoice(object key = null, string text = "", Color color = default)
    {
```

Reply:
I see. Not a fan, TBH.